fedimint-server/src/consensus/engine.rs

```diff
@@ -182,23 +187,44 @@ impl ConsensusEngine {
     }
 
     async fn confirm_server_config_consensus_hash(&self) -> anyhow::Result<()> {
-        let our_hash = self.cfg.consensus.consensus_hash();
-
-        info!(target: LOG_CONSENSUS, "Waiting for peers config {our_hash}");
+        info!(target: LOG_CONSENSUS, "Confirming server config consensus hash with peers...");
 
         loop {
             match self.federation_api.server_config_consensus_hash().await {
                 Ok(consensus_hash) => {
-                    if consensus_hash != our_hash {
+                    if consensus_hash != self.cfg.consensus.consensus_hash() {
                         bail!("Our consensus config doesn't match peers!")
                     }
 
-                    info!(target: LOG_CONSENSUS, "Confirmed peers config {our_hash}");
+                    info!(target: LOG_CONSENSUS, "Confirmed server config consensus hash with peers");
+
+                    return Ok(());
+                }
+                Err(e) => {
+                    warn!(target: LOG_CONSENSUS, "Could not confirm consensus config hash: {e}")
+                }
+            }
+
+            sleep(Duration::from_millis(100)).await;
+        }
+    }
```

Comment:
Some unrelated refactors, but looks good, especially not running it as a hot loop anymore :sweat_smile: